sidebars.js

```diff
@@ -41,6 +41,14 @@ const sidebars = {
             'user_docs/metrics/openrank/community_openrank'
           ],
         },
+        {
```

Comment:
Since `OpenRank` is a series algorithms for different scenarios, I add them under a directory, but for other metrics like `fork` are single metrics, so we can direct add them under metrics directory without another level.

Reply:
alright, I'll update it